src/libraries/System.DirectoryServices.Protocols/src/System/DirectoryServices/Protocols/common/BerConverter.cs

```diff
@@ -521,14 +501,9 @@ private static int EncodingMultiByteArrayHelper(BerSafeHandle berElement, byte[]
                         byte[] byteArray = tempValue[i];
 
                         // construct the managed berval
-                        managedBerVal[i] = new SafeBerval();
+                        managedBerVal[i] = new SafeBerval(0, IntPtr.Zero);
```

Comment:
Since it's a struct, and you're assigning it to the zero-value, you can just delete this line.  The array initializer already filled the array with zero-init values.